### PR TITLE
Update version: TGRMNSoftware.BulkRenameUtility version 3.4.4.0

### LINKED ISSUE #133397: [Update Request]: Bulk Rename Utility
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

TGRMNSoftware.BulkRenameUtility

### Package Version

3.4.4.0

### Please describe the changes you would like to see

I had an error "Installer Hash does not match".

Based on https://github.com/microsoft/winget-cli/issues/3640, I decided to create an issue here to make an update.

### PR DESCRIPTION
**Pull request by Exorcism0666**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.0.0 :rocket:

- Resolve: #133397

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/133403)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
image

**Comment by Exorcism0666** [comment]
image

**Comment by Exorcism0666** [comment]
image

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

File: manifests/t/TGRMNSoftware/BulkRenameUtility/3.4.4.0/TGRMNSoftware.BulkRenameUtility.installer.yaml [hunk 1]
```diff
@@ -1,23 +1,21 @@
...
-ManifestType: installer
-ManifestVersion: 1.2.0
+# Created with komac v2.0.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: TGRMNSoftware.BulkRenameUtility
+PackageVersion: 3.4.4.0
+MinimumOSVersion: 10.0.0.0
+InstallerLocale: en-US
+InstallerType: exe
```
Comment (OfficialEsco) [inline manifests/t/TGRMNSoftware/BulkRenameUtility/3.4.4.0/TGRMNSoftware.BulkRenameUtility.installer.yaml:8]:
```suggestion
InstallerType: inno
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**